backend/scandamus/scandamus/urls.py

```diff
@@ -7,11 +7,10 @@
     path('admin/', admin.site.urls),
     path('token/', TokenObtainPairView.as_view(), name='token_obtain_pair'),
     path('token/refresh/', TokenRefreshView.as_view(), name='token_refresh'),
-    path('api/players/', include('players.urls')),
-    path('api/friends/', include('players.urls')),
-    path('api/tournaments/', include('game.urls')),
-    path('api-internal/game/', include('game.urls')),
-    path('ws/lounge/', include('game.urls')),
+    path('api/players/', include('players.urls.players')),
+    path('api/friends/', include('players.urls.friends')),
+    path('api/tournaments/', include('game.urls.tournaments')),
+    path('api-internal/game/', include('game.urls.game')),
     path('health/', health_check),
-    path('api/test/', include('players.urls')), # for test use only
+    path('api/test/', include('players.urls.players')), # for test use only
```

Comment:
```suggestion
    path('api/test/', include('players.urls.test')), # for test use only
```
すいません！　追加でこちらを別ルールに分けないとでした
現状のplayers/urls/players.pyをコピーしてplayers/urls/test.pyとしておくと良さそうです